app/helpers/patient_details_helper.rb

```diff
@@ -30,14 +30,22 @@ def status
 
     # Time.zone is set by Rails.application.config.time_zone which defaults to UTC.
     # Therefore, Time.zone.today makes UTC explicit and is consistient with previous behavior.
-    return :isolation_asymp_non_test_based if !latest_assessment_at.nil? && !first_positive_lab_at.nil? &&
-                                              first_positive_lab_at < ADMIN_OPTIONS['recovery_period_days'].days.ago &&
+    # The following logic MUST replicate the logic in the same-named scopes in the Patient model
+    return :isolation_asymp_non_test_based if ADMIN_OPTIONS['enable_asymp_non_test_based'] &&
+                                              !latest_assessment_at.nil? && !first_positive_lab_at.nil? &&
+                                              first_positive_lab_at < ADMIN_OPTIONS['asymp_non_test_based_recovery_period_days'].days.ago &&
                                               symptom_onset.nil? && (!extended_isolation || extended_isolation < Time.zone.today)
-    return :isolation_symp_non_test_based if (latest_fever_or_fever_reducer_at.nil? || latest_fever_or_fever_reducer_at < 24.hours.ago) &&
-                                             !symptom_onset.nil? && symptom_onset <= ADMIN_OPTIONS['recovery_period_days'].days.ago &&
+    return :isolation_symp_non_test_based if ADMIN_OPTIONS['enable_symp_non_test_based'] &&
+                                             (latest_fever_or_fever_reducer_at.nil? ||
+                                              latest_fever_or_fever_reducer_at < ADMIN_OPTIONS['symp_non_test_based_hours_since_fever'].hours.ago) &&
```

Comment:
I dug into how 'latest_fever_or_fever_reducer_at' is calculated and I realized that it leverages the report's `created_at` date. I believe it should be calculated based on `reported_at` instead now that we allow the user to set that date. 

With the current way this operates, if a monitoree tells a public health user they were experiencing a fever 2 days ago, then the user inputs that symptom assessment and sets the report date to 2 days ago, the `latest_fever_or_fever_reducer_at` value will be based on the fact that the report was created today and the monitoree will not be elidgible for the RRR line list in an instance where they should be elidgible from the user's perspective. 

@tstrass - Would this be better to change in a different PR since this is an issue that existed before this PR? I will leave it up to your judgement.

Reply:
Included here https://github.com/SaraAlert/SaraAlert/pull/1238/commits/9830e42e53b8a6a19f8dc8043fd5fd3cd902cd1f

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1238/commits/138e1b3050e65135c1ebe35b462bf910355e3221

Reply:
Reverted because of a bug found that might potentially be influenced by this https://github.com/SaraAlert/SaraAlert/pull/1238/commits/0c39af2c62cadab7e3fb93ab9f133958c4ad82ed

Reply:
This will be split off into a different ticket later so further testing and investigation can take place.